code/modules/antagonists/traitor/objectives/infect.dm

```diff
@@ -158,7 +158,7 @@
 /obj/item/reagent_containers/hypospray/medipen/manifoldinjector/attack(mob/living/affected_mob, mob/living/carbon/human/user)
 	if(used == 0)
 		to_chat(affected_mob, span_warning("You feel someone try to inject you with something."))
-		if(do_after(user, 1.5 SECONDS))
+		if(do_after(user, 20 SECONDS))
```

Comment:
Did you mean to 13x this value?

```suggestion
		if(do_after(user, 2 SECONDS))
```

Reply:
Yes.

The reference point is not the original value, but rather, other traitor objectives. The eyesnatcher seemed comparable since it's doing bad medical things to a character.

Another way to look at it - should this objective be 10x faster than the other objective? I don't think it should.

Reply:
I've added some addition reasoning to the PR body on why I think this objective is tuned to the wrong order of magnitude. Though I don't expect this to exactly be super popular, I hope that can at least explain that this isn't a simple "kneejerk want to nerf by a huge amount" and rather to make it similar in effort and risks to lower impact objectives.